mslib/msui/msui_mainwindow.py

```diff
@@ -36,6 +36,7 @@
 import os
 import re
 import sys
+import fs
```

Comment:
there is a problem with the rebase to the current develop. 
fs was removed and completly replaced by pathlib.Path

Reply:
I see the problem by
```
 python mslib/msui/msui.py
Traceback (most recent call last):
  File "/Users/reimarbauer/PycharmProjects/2025/annapurna-gupta/MSS/mslib/msui/msui.py", line 40, in <module>
    from mslib.msui.msui_mainwindow import MSUIMainWindow
  File "/Users/reimarbauer/PycharmProjects/2025/annapurna-gupta/MSS/mslib/msui/msui_mainwindow.py", line 39, in <module>
    import fs
ModuleNotFoundError: No module named 'fs'
```